repos/fdbt-site/src/pages/api/pointToPointPeriodProduct.ts

```diff
@@ -62,7 +63,12 @@ export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
         }
 
         updateSessionAttribute(req, POINT_TO_POINT_PRODUCT_ATTRIBUTE, pointToPointPeriodProduct);
-        redirectTo(res, '/periodValidity');
+
+        if (globalSettingsEnabled) {
+            redirectTo(res, '/globalSettingsPeriodValidity');
+        } else {
+            redirectTo(res, '/periodValidity');
+        }
```

Comment:
Think there are a few other places that direct to this page that need changing? Alternatively you could direct from one page to the other?
![image](https://user-images.githubusercontent.com/33119300/132683520-81615e07-faf3-4b3a-90f9-442fe733e88e.png)

Reply:
What about the chooseValidity one @JwanKhalaf ? Does that even get used anywhere now? Guess we need to either update or delete it

Reply:
No, I can't see anything using `chooseValidity`! I'll delete it.